src/flake8_bugbear/plugins/getattr_with_constant.rs

```diff
@@ -1,30 +1,57 @@
 use once_cell::sync::Lazy;
 use regex::Regex;
-use rustpython_ast::{Constant, Expr, ExprKind};
+use rustpython_ast::{Constant, Expr, ExprContext, ExprKind};
 
 use crate::ast::types::Range;
+use crate::autofix::Fix;
 use crate::check_ast::Checker;
 use crate::checks::{Check, CheckKind};
+use crate::code_gen::SourceGenerator;
 use crate::python::keyword::KWLIST;
 
 static IDENTIFIER_REGEX: Lazy<Regex> =
     Lazy::new(|| Regex::new(r"^[A-Za-z_][A-Za-z0-9_]*$").unwrap());
 
+fn attribute(value: &Expr, attr: String) -> Expr {
+    Expr::new(
+        Default::default(),
+        Default::default(),
+        ExprKind::Attribute {
+            value: Box::new(value.clone()),
+            attr,
+            ctx: ExprContext::Load,
+        },
+    )
+}
+
 /// B009
 pub fn getattr_with_constant(checker: &mut Checker, expr: &Expr, func: &Expr, args: &[Expr]) {
     if let ExprKind::Name { id, .. } = &func.node {
         if id == "getattr" {
-            if let [_, arg] = args {
+            if let [obj, arg] = args {
```

Comment:
What happens if we have, like:

```py
getattr(x, r"y")
```

(That is, if the string has a kind or prefix code?)

Reply:
Actually, think I understand this. Will add some tests and merge.

Reply:
Thanks for the update!